internal/resources/azure/automation_account.go

```diff
@@ -0,0 +1,84 @@
+package azure
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+
+	"fmt"
+
+	"github.com/shopspring/decimal"
+)
+
+type AutomationAccount struct {
+	Address                 string
+	Region                  string
+	MonthlyJobRunMins       *int64 `infracost_usage:"monthly_job_run_mins"`
+	NonAzureConfigNodeCount *int64 `infracost_usage:"non_azure_config_node_count"`
+	MonthlyWatcherHours     *int64 `infracost_usage:"monthly_watcher_hrs"`
```

Comment:
```suggestion
	MonthlyWatcherHrs     *int64 `infracost_usage:"monthly_watcher_hrs"`
```
Just to keep it consistent